nrespipe/trace_orders.py

```diff
@@ -0,0 +1,1090 @@
+"""
+Written by Allen B. Davis (c) 2018.
+Yale University.
+
+Last update: May 2, 2018.
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.signal import medfilt
+import collections
+import warnings
+import sys
+import datetime
+from astropy.io import fits
+
+
+@app.task
```

Comment:
app is not defined in this scope.

Reply:
I've never used celery, so I'm not sure how to fix this issue. I didn't have a chance to get to this part while I was visiting LCO.

Reply:
I would recommend moving the trace routine to the tasks.py. The app variable should be imported there already.

Reply:
To be clear, you mean move the function `trace` (which calls various functions from trace_orders.py) from the trace_orders.py to tasks.py? In this case, I would need to import trace_orders in tasks.py. Is that OK?

Reply:
And should I keep the docstring if I move the function over? None of the other tasks.py functions have docstrings.

Reply:
Moved `trace` to tasks.py. Changed name of trace_orders.py to trace_orders_utils.py, since it now contains only subroutines. Imported this in tasks.py.